src/template.html

```diff
@@ -4,6 +4,7 @@
 	<meta charset='utf-8'>
 	<meta name='viewport' content='width=device-width,initial-scale=1.0'>
 	<meta name='theme-color' content='#333333'>
+	<script src="https://cdn.jsdelivr.net/gh/microsoft/TypeScript/lib/typescriptServices.js" async></script>
```

Comment:
Is there a way to get a versioned URL in jsdeliver? If not, I'd prefer if we used a taggd version, like https://raw.githubusercontent.com/microsoft/TypeScript/v3.9.5/lib/typescriptServices.js